.github/workflows/deploy.yml

```
@@ -7,20 +7,27 @@ on:
 jobs:
   test:
     uses: "ambanum/OpenTermsArchive/.github/workflows/test.yml@main"
+
+  get_instances:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set matrix with instance names
```

This step does not actually set any matrix :wink:

```suggestion
      - name: Get all declared instances
```